[2021-09-13 06:56:02,685] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: Homework-task2.row_count 2021-09-12T00:00:00+00:00 [queued]>
[2021-09-13 06:56:02,798] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: Homework-task2.row_count 2021-09-12T00:00:00+00:00 [queued]>
[2021-09-13 06:56:02,817] {taskinstance.py:1094} INFO - 
--------------------------------------------------------------------------------
[2021-09-13 06:56:02,824] {taskinstance.py:1095} INFO - Starting attempt 1 of 6
[2021-09-13 06:56:02,826] {taskinstance.py:1096} INFO - 
--------------------------------------------------------------------------------
[2021-09-13 06:56:02,970] {taskinstance.py:1114} INFO - Executing <Task(BashOperator): row_count> on 2021-09-12T00:00:00+00:00
[2021-09-13 06:56:03,023] {standard_task_runner.py:52} INFO - Started process 27893 to run task
[2021-09-13 06:56:03,076] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'Homework-task2', 'row_count', '2021-09-12T00:00:00+00:00', '--job-id', '450', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/task_2.py', '--cfg-path', '/tmp/tmplrvt4pcv', '--error-file', '/tmp/tmpi5_r00x6']
[2021-09-13 06:56:03,092] {standard_task_runner.py:77} INFO - Job 450: Subtask row_count
[2021-09-13 06:56:03,524] {logging_mixin.py:109} INFO - Running <TaskInstance: Homework-task2.row_count 2021-09-12T00:00:00+00:00 [running]> on host ed2b66e7bf44
[2021-09-13 06:56:03,938] {taskinstance.py:1251} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=Homework-task2
AIRFLOW_CTX_TASK_ID=row_count
AIRFLOW_CTX_EXECUTION_DATE=2021-09-12T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-09-12T00:00:00+00:00
[2021-09-13 06:56:03,942] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2021-09-13 06:56:04,053] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'wc -l < $AIRFLOW_HOME/data/downloaded_file.csv']
[2021-09-13 06:56:04,277] {subprocess.py:74} INFO - Output:
[2021-09-13 06:56:04,452] {subprocess.py:78} INFO - 3061
[2021-09-13 06:56:04,469] {subprocess.py:82} INFO - Command exited with return code 0
[2021-09-13 06:56:04,653] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=Homework-task2, task_id=row_count, execution_date=20210912T000000, start_date=20210913T065602, end_date=20210913T065604
[2021-09-13 06:56:04,964] {local_task_job.py:151} INFO - Task exited with return code 0
[2021-09-13 06:56:05,671] {local_task_job.py:261} INFO - 0 downstream tasks scheduled from follow-on schedule check
